### PR TITLE
Words ui

### PR DESCRIPTION
**Pull request by kiennt175**
## Related Tickets:
- [#31939](https://edu-redmine.sun-asterisk.vn/issues/31939)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kiennt175** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
2 commits

**Comment by kiennt175** [comment]
ready

File: app/Http/Controllers/WordController.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            ['type_id', '=', $type_id],
+        ])->get()->isNotEmpty();
+        if ($check_word) 
+            return back()->with('message', trans('words.check_word'));
+        try {
+            $word = Word::find($word_id);
+            $types = $word->types()->get()->toArray();
+            $totalRecords = count($types);
+            $word->types()->wherePivot('type_id', $type_id)->detach();
+            if ($totalRecords === 1) {
```
Comment (huyenttm-0901) [inline app/Http/Controllers/WordController.php]: đưa số vào config

File: app/Http/Controllers/WordController.php [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                $word->delete();
+            }
+        } catch (Exception $e) {
+            return redirect()->route('words.index')->with('message', trans('words.delete_failed'));
+        }
+
+        return redirect()->route('words.index')->with('message', trans('words.delete_successfully'));
+    }
+
+    public function fix($word_id, $type_id)
```
Comment (huyenttm-0901) [inline app/Http/Controllers/WordController.php]:
```suggestion
    public function fix($wordId, $typeId)
```

**Comment by kiennt175** [comment]
ready

**Comment by kiennt175** [comment]
ready

**Comment by kiennt175** [comment]
ready

**Pull request merged**